deepinterpolation/inferrence_collection.py

```diff
@@ -1,13 +1,24 @@
 import warnings
 
 import h5py
+import logging
 import numpy as np
 from deepinterpolation.generic import JsonLoader
+from deepinterpolation.generator_collection import DeepGenerator
 from tensorflow.keras.models import load_model
 import deepinterpolation.loss_collection as lc
 from tqdm.auto import tqdm
-
-
+import tensorflow as tf
+import multiprocessing
+from multiprocessing.managers import DictProxy, AcquirerProxy
+from deepinterpolation.multiprocessing_utils import _winnow_process_list
```

Comment:
importing private function from module outside package. Should make this function public then.